### PR TITLE
Update object store submodule pointer

### PR DESCRIPTION
**Pull request by austinzheng**
Would appreciate any corrections or additions required to the changelog notes.

### CODE REVIEW

File: dependencies.list [hunk 1]
```diff
@@ -1,4 +1,4 @@
 VERSION=2.4.3
-REALM_CORE_VERSION=2.3.1
-REALM_SYNC_VERSION=1.1.0
+REALM_CORE_VERSION=2.3.2
+REALM_SYNC_VERSION=1.3.0
```
Comment (bdash) [inline dependencies.list]: Are there any other bug fixes in these two components that we should call out in our change log?

Comment (austinzheng) [inline dependencies.list]: Probably. I'll go through the changelogs when I get home and update the PR.

Comment (austinzheng) [inline dependencies.list]: It actually looks like there are no public-facing changes for sync, and the core changes are both for Android issues.

Comment (bdash) [inline dependencies.list]: realm/realm-core#2465 doesn't appear to be an Android-specific fix?

Comment (austinzheng) [inline dependencies.list]: That should probably be mentioned.

**Pull request merged**